lib/shopify_api.rb

```diff
@@ -32,3 +32,8 @@ module ShopifyAPI
 else
   require 'active_resource/connection_ext'
 end
+
+if ENV["SHOPIFY_LOG"] == '1'
+  ActiveResource::Base.logger = Logger.new("shopify_api.log")
+  ActiveResource::DetailedLogSubscriber
```

Comment:
Hmm, does referencing this without calling a method still have an affect?

Reply:
I'm able to get this working locally. I assumed that the entire file was executed

Reply:
Hmm, this may be a side-effect of the autoloading. Let me check it locally...

Reply:
Ok, so `DetailedLogSubscriber` currently does this:

https://github.com/Shopify/shopify_api/blob/master/lib/active_resource/detailed_log_subscriber.rb#L53

I think we should move this line out of `DetailedLogSubscriber` and into our logging setup code.

Otherwise it's not intuitive that loading the class will enable logging as this does.

cc @Shopify/platform-dev-tools-education for a second opinion, as the code has been this way for a [very long time](https://github.com/Shopify/shopify_api/commit/5b865286e83074e3020d0b3b36590b158368cf6f).